addons/web_editor/static/lib/odoo-editor/src/OdooEditor.js

```diff
@@ -1175,6 +1175,27 @@ export class OdooEditor extends EventTarget {
         }
     }
 
+    linkInsertionFixActivate(link) {
```

Comment:
If we are going to have such a function, then I think it would be best to have its opposite as a function as well, to be able to easily track where the effects of this function is reverted, even though it is only called once.

Reply:
What do you mean? The function `_resetContenteditableLinks` already exits.

Reply:
Then we might want to find another name for this function so that we can easily understand the link that exists between the two. I suggest to rename `linkInsertionFixActivate` to `_setContenteditableLinks` so that we have the "set" and the "reset" functions. What do you think ?

Reply:
You mean `setContenteditableLinks` rather than `_setContenteditableLinks` ?
We are using this function outside the class.

Reply:
Isn't it a bit weird that one is public but not the other ?

Reply:
I'm fine having `_resetContenteditableLinks` renamed to `resetContenteditableLinks`.
My question is about calling renaming `linkInsertionFixActivate` to `setContenteditableLinks` rather than `_setContenteditableLinks` (as you suggested).

Reply:
I suggested `_setContenteditableLinks` because I mirrored `_resetContenteditableLinks`. I'm fine having `setContenteditableLinks` and `resetContenteditableLinks`. It must not start with an `_` if the function is called from outside the class indeed.